.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "airbnb",
+  "env": {
+    "mocha": true
+  },
+  "rules": {
+    "semi": [2, "never"],
+    "no-param-reassign": 0
```

Comment:
We are mutating [traversedPath.node.leadingComments](https://github.com/cef62/css-modules-compiler/blob/master/src/update-css-modules-imports.js#L46) and I usually avoid polluting the code with eslint-related comments.
Do you prefer me to disable the rule for that line only?